direct/io-kafka/src/test/java/cz/o2/proxima/direct/kafka/LocalKafkaCommitLogDescriptor.java

```diff
@@ -449,11 +449,11 @@ private void seekConsumerToBeginning(ConsumerId consumerId, Collection<TopicPart
     }
 
     private <K, V> ConsumerRecords<K, V> pollConsumer(
-        ConsumerGroup group,
-        long period,
-        ConsumerId consumerId,
-        ElementSerializer<K, V> serializer,
-        @Nullable ConsumerRebalanceListener listener)
+        final ConsumerGroup group,
```

Comment:
why this change?

Reply:
You mean the `final`? The `consumerId` is used for synchronization inside the method, so it should be final, and when doing that I made all parameters final for consistency.